### ISSUE TITLE
Bulk add cards

### ISSUE THREAD

**Issue by putt1ck** [nextcloud/deck#1104]
When you have an existing list of items (sent to you, or copied from a meetings minutes etc.) and want to add them, it would be great if you could paste them into a card and if the pasted content contains new line markers you got an option to have each new line created as new cards.

Related: if the pasted content contains new lines, the alternative to a new cards would be first line as card title and rest added into the description field (on the basis that new lines don't make sense in a title and aren't supported anyway)

**Comment by jakobroehrl** [comment]
@juliushaertl How can I detect a line break in the card title here:
https://github.com/nextcloud/deck/blob/vue/src/components/board/Stack.vue#L134-L141
`this.newCardTitle.match/\r\n/` isn't working, thank you!

**Comment by juliushaertl** [comment]
@jakobroehrl This won't work as the text input doesn't support multiple lines and I think we should keep that.

I'm also not sure about automatically detecting the paste of multiple lines and creating different cards for it, since this might be unexpected behavior to users who just want to create a card from something copied. cc @nextcloud/designers

**Comment by stefan-niedermann** [comment]
Maybe just in a special case, e.g. pasting a bunch of lines starting with `- ` or `* `? Or maybe even more special, starting with `- [ ]` or `- [x]`?

**Comment by jakobroehrl** [comment]
Bulk edit Button changes textfield in textarea

**Comment by jancborchardt** [comment]
> I'm also not sure about automatically detecting the paste of multiple lines and creating different cards for it, since this might be unexpected behavior to users who just want to create a card from something copied. cc @nextcloud/designers

Agree. If anything, pasting a bunch of stuff I’d expect it to create one card with the rest in details, as was also described above.

We should definitely _not_ do both "modes" as it will lead us down a path of eternal confusion and damnation.

**Comment by juliushaertl** [comment]
> Bulk edit Button changes textfield in textarea

I also think that the use case is to specific to justify an additional button.

**Comment by putt1ck** [comment]
What if the new line detection was in the title? Paste a list of items into a new card title and for every new line detected an additional card is added
